src/main/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpoint.java

```diff
@@ -86,6 +91,11 @@ public GlobalSearchEndpoint(@Qualifier(AuthorisedCaseSearchOperation.QUALIFIER)
             })
         )
     })
+    @LogAudit(
+        operationType = AuditOperationType.GLOBAL_SEARCH,
+        caseTypeIds = "T(uk.gov.hmcts.ccd.endpoint.std.GlobalSearchEndpoint).buildCaseTypeIds(#result)",
```

Comment:
`CaseSearchEndpoint` logs the CaseTypeIds from the input not the response.  Should this be doing the same?